data/projects/implement_metrics_for_coala.md

```diff
@@ -7,28 +7,46 @@ them can be measured objectively by so-called
 offers a good base for the implementation or wrapping of algorithms that
 calculate these metrics from a piece of code.
 
+coala should have a module that can produce software metrics because software
+metrics are crucial when it comes to quality assurance. Source code metrics
+for quality assurance aims at writing code with less bugs inside and writing
+code for easy maintenance. Both lead to writing code as simple as possible
+with few elements in each unit. Picking right metrics is paramount and this
+project aims at implementing an API and a set of Bears for producing
+software metrics.
+
 The goal of this project proposal should be to implement handling and
 display of software metrics in coala. To prove the concept, this project
 has to include multiple bears that calculate metrics for different
 programming languages.
 
+
 #### Milestones
 
-##### GSOC 2017 COMMUNITY BONDING
+##### PREPARATION/BONDING
 
+* Research existing tools for software metrics.
```

Comment:
Make milestones testable and rather say "A list of software metric tools to support exists" because this is a state that can easily be checked against